apis/wikis.es6.js

```diff
@@ -0,0 +1,31 @@
+import { runQuery } from '../apiBase/APIRequestUtils';
+import Wiki from '../models2/Wiki';
+
+const getPath = (query) => {
```

Comment:
🔕 we should get into the habit of `export`ing these functions and writing tests for them